plugins/github-pull-requests-board/src/hooks/useTeamRepositories.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useApi } from '@backstage/core-plugin-api';
+import {
+  catalogApiRef,
+  humanizeEntityRef,
+  useEntity,
+} from '@backstage/plugin-catalog-react';
+import { useCallback, useEffect, useState } from 'react';
+import { getProjectNameFromEntity } from '../utils/functions';
+
+export function useTeamRepositories() {
+  const { entity: teamEntity } = useEntity();
+  const catalogApi = useApi(catalogApiRef);
+  const [repositories, setRepositories] = useState<string[]>([]);
+
+  const getRepositoriesNames = useCallback(async () => {
+    const entitiesList = await catalogApi.getEntities({
+      filter: {
+        'spec.owner': humanizeEntityRef(teamEntity, { defaultKind: 'group' }),
```

Comment:
should generally use `stringifyEntityRef` instead, except in some rare "only display to human eyes" scenarios

Reply:
Oh and also, this should not filter on `spec.owner`, it should filter on `relations.ownedBy`. Spec is only one of many possible drivers for the generation of relations, and relations are the only thing that should normally be consumed if it's relations that one is looking for. Otherwise you'll miss all the entities that have their ownership set through other means than the spec.